earth2studio/models/px/fcn3.py

```diff
@@ -0,0 +1,357 @@
+
+VARIABLES = [
+    "u10m",
+    "v10m",
+    "u100m",
+    "v100m",
+    "t2m",
+    "msl",
+    "tcwv",
+    "u50",
+    "u100",
+    "u150",
+    "u200",
+    "u250",
+    "u300",
+    "u400",
+    "u500",
+    "u600",
+    "u700",
+    "u850",
+    "u925",
+    "u1000",
+    "v50",
+    "v100",
+    "v150",
+    "v200",
+    "v250",
+    "v300",
+    "v400",
+    "v500",
+    "v600",
+    "v700",
+    "v850",
+    "v925",
+    "v1000",
+    "z50",
+    "z100",
+    "z150",
+    "z200",
+    "z250",
+    "z300",
+    "z400",
+    "z500",
+    "z600",
+    "z700",
+    "z850",
+    "z925",
+    "z1000",
+    "t50",
+    "t100",
+    "t150",
+    "t200",
+    "t250",
+    "t300",
+    "t400",
+    "t500",
+    "t600",
+    "t700",
+    "t850",
+    "t925",
+    "t1000",
+    "q50",
+    "q100",
+    "q150",
+    "q200",
+    "q250",
+    "q300",
+    "q400",
+    "q500",
+    "q600",
+    "q700",
+    "q850",
+    "q925",
+    "q1000",
+]
+
+
+@check_optional_dependencies()
+class FCN3(torch.nn.Module, AutoModelMixin, PrognosticMixin):
+    """
+
+    FourCastNet 3 global probabilisticprognostic model.
+    Consists of a single model with a time-step size of 6 hours.
+    FourCastNet 3 operates on 0.25 degree lat-lon grid (south-pole excluding)
+    equirectangular grid with 72 variables.
+
+    Parameters
+    ----------
+    core_model : torch.nn.Module
+        Core PyTorch model with loaded weights
+    center : torch.Tensor
+        Model center normalization tensor
+    scale : torch.Tensor
+        Model scale normalization tensor
+    mins : torch.Tensor
+        Model minimum normalization tensor
+    maxs : torch.Tensor
+        Model maximum normalization tensor
+    variables : np.array, optional
+        Variables associated with model, by default 73 variable model.
+    """
+
+    def __init__(
+        self,
+        core_model: torch.nn.Module,
+        variables: np.array = np.array(VARIABLES),
+    ):
+        super().__init__()
+        self.model = core_model
+        self.variables = variables
+        if "2d" in self.variables:
+            self.variables[self.variables == "2d"] = "d2m"
+
+    def __str__(self) -> str:
+        return "fcn3"
+
+    def input_coords(self) -> CoordSystem:
+        """Input coordinate system of the prognostic model
+        Returns
+        -------
+        CoordSystem
+            Coordinate system dictionary
+        """
+        return OrderedDict(
+            {
+                "batch": np.empty(0),
+                "time": np.empty(0),
+                "lead_time": np.array([np.timedelta64(0, "h")]),
+                "variable": np.array(self.variables),
+                "lat": np.linspace(90.0, -90.0, 721),
+                "lon": np.linspace(0, 360, 1440, endpoint=False),
+            }
+        )
+
+    @batch_coords()
+    def output_coords(self, input_coords: CoordSystem) -> CoordSystem:
+        """Output coordinate system of the prognostic model
+        Parameters
```

Comment:
Nit add spaces between sections in doc strings

Reply:
Added better docstrings.